sample-bchd.conf

```diff
@@ -273,13 +348,21 @@
 ; transactions available via the getrawtransaction RPC.
 ; txindex=1
 
+; Deletes the hash-based transaction index from the database on start up
+; and then exits.
+; droptxindex=1
```

Comment:
I think both drop indexes can probably be removed from here as someone would likely only want to run these options just once rather than on every startup.

Reply:
Will do. There are three drop index options by the way: `droptxindex`, `dropaddrindex` and `dropcfindex`. I'm removing all three from the sample configuration.